### ISSUE TITLE
SimpleReport logo on static site pages has incorrect alt text

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4037]
## Area or Component
Static site, support/help pages

## Quick Summary
The logo in the top left of the page has the alt text "SimpleReport" but in this case it's better described as "SimpleReport logo". Also, it's preceded by a hidden header with exactly the same text.

## Accessibility Violation
Alternative text of images should not be repeated as text: https://dequeuniversity.com/rules/axe/4.4/image-redundant-alt?application=AxeChrome

## Action Requested
Either place the hidden header around the image, change the image alt text to include logo, or change the DOM itself to have a visible H1 with the text "SimpleReport" and the circle logo as a separate image element. (The last one would be best if possible)

## Screenshots (optional)
Logo/image in the top left of the screen:
![Screen Shot 2022-07-21 at 5 59 35 PM](https://user-images.githubusercontent.com/80282552/180339367-efb61c19-de62-454d-baef-956f88fb8cf5.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1510420308&range=10:10
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1510420308&range=6:6
